Cannot create instance (default settings)

Hello Team !

exo version: `exo 1.51.1 6fbf2691 (egoscale 0.86.0)`

When I try to create an instance with default params `exo compute instance create my-test-instance`, I got an error message (but the instance is available):

panic: reflect: NumField of non-struct type string

goroutine 1 [running]:
reflect.(*rtype).NumField(0x30)
        reflect/type.go:1015 +0x66
github.com/exoscale/cli/cmd.outputTableHeaders({0x1765c10, 0x10520a0})
        github.com/exoscale/cli/cmd/output.go:140 +0x45
github.com/exoscale/cli/cmd.outputTable({0x1103180, 0xc000622a20})
        github.com/exoscale/cli/cmd/output.go:271 +0x938
github.com/exoscale/cli/cmd.(*instanceShowOutput).toTable(0xc000618490)
        github.com/exoscale/cli/cmd/instance_show.go:34 +0x25
github.com/exoscale/cli/cmd.output({0x1730368, 0xc000622a20}, {0x0, 0x0})
        github.com/exoscale/cli/cmd/output.go:59 +0x84
github.com/exoscale/cli/cmd.(*instanceShowCmd).cmdRun(0xc00077fc68, 0x173ffe0, {0xc000898ae0, 0x7ffe9c04c658, 0x7})
        github.com/exoscale/cli/cmd/instance_show.go:163 +0x943
github.com/exoscale/cli/cmd.(*instanceCreateCmd).cmdRun(0xc0000e11d0, 0xc00062fd00, {0xc00062fcd8, 0x0, 0x0})
        github.com/exoscale/cli/cmd/instance_create.go:238 +0x16f9
github.com/spf13/cobra.(*Command).execute(0xc00028d900, {0xc000898330, 0x3, 0x3})
        github.com/spf13/cobra@v1.1.3/command.go:852 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x1ed2fa0)
        github.com/spf13/cobra@v1.1.3/command.go:960 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.1.3/command.go:897
github.com/exoscale/cli/cmd.Execute({0x17075f0, 0x419025}, {0x1707d98, 0xc0000001a0})
        github.com/exoscale/cli/cmd/root.go:102 +0x21f
main.main()
        github.com/exoscale/cli/main.go:34 +0x46


@pyr : our last release is affected, I think.

--- a/cmd/output.go
+++ b/cmd/output.go
@@ -266,17 +266,26 @@ func outputTable(o interface{}) {
 			if n := v.Field(i).Len(); n == 0 {
 				tab.Append([]string{label, "n/a"})
 			} else {
-				var embeddedBuf bytes.Buffer
-				embeddedTable := table.NewEmbeddedTable(&embeddedBuf)
-				embeddedTable.SetHeader(outputTableHeaders(v.Field(i).Type().Elem()))
+				switch v.Field(i).Type().Elem().Kind() {
+				case reflect.Struct:
+					var embeddedBuf bytes.Buffer
+					embeddedTable := table.NewEmbeddedTable(&embeddedBuf)
 
-				for j := 0; j < reflect.Indirect(v.Field(i)).Len(); j++ {
-					row := outputTableRow(reflect.Indirect(v.Field(i)).Index(j))
-					embeddedTable.Append(row)
+					embeddedTable.SetHeader(outputTableHeaders(v.Field(i).Type().Elem()))
+
+					for j := 0; j < reflect.Indirect(v.Field(i)).Len(); j++ {
+						row := outputTableRow(reflect.Indirect(v.Field(i)).Index(j))
+						embeddedTable.Append(row)
+					}
+
+					embeddedTable.Render()
+					tab.Append([]string{label, embeddedBuf.String()})
+				case reflect.String:
+					items := v.Field(i).Interface().([]string)
+					tab.Append([]string{label, strings.Join(items, "\n")})
+				default:
+					tab.Append([]string{label, "(type not supported)\n"})
 				}
-
-				embeddedTable.Render()
-				tab.Append([]string{label, embeddedBuf.String()})
 			}
 
 		case reflect.Map:

